packages/common-all/src/analytics.ts

```diff
@@ -47,8 +47,16 @@ export enum TutorialEvents {
   Tutorial_3_Show = "Tutorial_3_Show",
   Tutorial_4_Show = "Tutorial_4_Show",
   Tutorial_5_Show = "Tutorial_5_Show",
+  TutorialNoteViewed = "TutorialNoteViewed",
 }
 
+export type TutorialNoteViewedPayload = {
+  testGroup: string;
```

Comment:
subtle distinction, but rather than 'testGroup', I think we should be tracking something called 'tutorialType'